healthcare/v1beta1/src/test/java/snippets/healthcare/FhirResourceTests.java

```diff
@@ -235,9 +236,25 @@ public void test_FhirResourceGetHistory() throws Exception {
     FhirResourcePatch.fhirResourcePatch(
         fhirResourceName,
         "[{\"op\": \"add\", \"path\": \"/active\", \"value\": false}]");
-    FhirResourceGetHistory.fhirResourceGetHistory(fhirResourceName);
+    // Get versionId from results of fhirResourcePatch.
+    String versionId;
+    Matcher idMatcher = Pattern.compile("\"versionId\": \"(.*)\"").matcher(bout.toString());
```

Comment:
This is fine for tests, but in samples, I'd prefer we used json objects using Guava's `GSON`, Jackson, or `org.json` `JAVA-Json`